backend/src/main/java/ca/mcgill/purposeful/model/VerificationRequest.java

```diff
@@ -15,7 +15,7 @@ public class VerificationRequest {
   public enum Status {
     Pending,
     Approved,
-    Refused
```

Comment:
Thanks for this change! Now that the enum is being used by 2 classes, does it still make sense to keep it here, or should we define move it somewhere else for quick access? I don't mind either way, just pointing it out incase

Reply:
It doesn't really change anything functionality-wise, but it would be clearer in a separate file